CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift

```diff
@@ -16,12 +16,8 @@ class OnboardingViewController: UIViewController {
     private var navigationBarBottomAdapter: OnboardingNavigationBarBottomAdapter?
     private var bottomNavigationBar: UIView?
 
-    lazy var skipButton = UIBarButtonItem(title: NSLocalizedStringPreferredFormat(
-        "ginicapture.onboarding.skip",
-        comment: "Skip button"),
-                                          style: .plain,
-                              target: self,
-                              action: #selector(close))
+    lazy var skipButton = GiniBarButton(ofType: .skip)
```

Comment:
[suggestion] This property can be private.

Reply:
Thank you, I made it private.